src/System.Private.ServiceModel/tools/setupfiles/sync-pr.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+show_banner()
+{
+    echo ""
+    echo "WCF Remote Github Repo sync script"
+}
+
+show_usage() 
+{
+    echo "    A URL must be specified for the pull request synchronization URL"
+    echo "    Usage: $0 [sync-url]"
+    echo "    sync-url - URL on remote server for PR synchronization"
+    echo ""
+    echo "    Example:  $0 http://wcfci-sync-server/repo/sync.ashx"
+}
+
+run_request() 
+{
+    __sync_url=$1
+    __ghprbPullId=$2
+
+    __request_uri=${__sync_url}?pr=${__ghprbPullId}
+
+    echo "Making a call to '${__request_uri}'" 
+
+    if [ ! -e "$__nugetpath" ]; then
+        which curl wget > /dev/null 2> /dev/null
+        if [ $? -ne 0 -a $? -ne 1 ]; then
+            echo "cURL or wget is required to make this request. Please see https://github.com/dotnet/wcf/blob/master/Documentation/building/unix-instructions.md for more details."
+            exit 1
+        fi
+
+        # curl has HTTPS CA trust-issues less often than wget, so lets try that first.
+        which curl > /dev/null 2> /dev/null
+        if [ $? -ne 0 ]; then
+           wget -q -O- ${__sync_url}?pr=${__ghprbPullId}
+        else
+           output=`curl -sw "\n%{http_code}" ${__sync_url}?pr=${__ghprbPullId}`
+           echo $output
+           retCode=`echo ${output} | grep "200\$"`
```

Comment:
I have to do this kind of calisthenics here since curl always returns a 0 error code as long as a response was received by the server, regardless of the server's response code

The way this CURL outputs is that the HTTP return code always sits at the end of the returned string, for example: 

"Syncing to PR = 901... OK 200"
"Bad Request 400"

The grep string will look for a 200 at the end. 

Note that wget will fail with exit code 1 when the server returns an error code